security.sh

```diff
@@ -3,11 +3,13 @@
 export LC_ALL=C.UTF-8
 export LANG=C.UTF-8
 export PYTHONDONTWRITEBYTECODE=1
-
 zap-api-scan.py -t ${URL_FOR_SECURITY_SCAN}/v2/api-docs -f openapi -S -d -u ${SECURITY_RULES} -P 1001 -l FAIL --hook=zap_hooks.py
-cat zap.out
```

Comment:
why it is removed, does it tell us what went wrong?

Reply:
replaced